package.json

```diff
@@ -10,7 +10,7 @@
     "start": "node tools/run.js",
     "build": "node tools/build.js",
     "build:debug": "node tools/build.js --debug",
-    "build:lib": "node tools/lib.build.js",
+    "build:lib": "node tools/lib.build.js && flow-copy-source -i '**/@(__mocks__|__tests__)/**' ./src ./dist && mv ./dist/lib.js.flow ./dist/index.js.flow",
```

Comment:
Why do we have to rename lib.js.flow to index.js.flow? Can we change the name in the webpack script?

Reply:
Webpack builds the library into index.js which is necessary to distribute the library via npm package (you could theoretically adjust the default location of you library-root-file in the package.json, but then IDEs have a hard time resolving it -- at least when I did that the last time Intellij had no clue what was going on ^^)